libbindgen/src/ir/item.rs

```diff
@@ -1004,19 +1005,35 @@ impl ClangItemParser for Item {
                         assert_eq!(popped_decl, declaration_to_look_for);
                     }
 
-                    location.visit(|cur| {
+                    fn visit_child(cur: clang::Cursor,
```

Comment:
Let's pull this function out of its enclosing parent, because it's getting busy and this isn't closing over anything.